dwave/plugins/torch/autoencoder/losses/kl_divergence.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2025 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from dimod import Sampler
+
+from dwave.plugins.torch.boltzmann_machine import AbstractBoltzmannMachine
```

Comment:
Would need to update this to `GraphRestrictedBoltzmannMachine` due to the [other PR](https://github.com/dwavesystems/dwave-pytorch-plugin/pull/14).